### DISCUSSION TITLE
Account suspended after using Co-Pilot free trial

### DISCUSSION THREAD

**Discussion by thedevinc** [community/community#135569] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone. I recently tested GitHub co-pilot with the free trial and I was unable to use it the entire time. I then went ahead to proceed with canceling the free trial and noticed my account got suspended on my company account (Account: dAppleganger) . I just wanted to cancel it because I was unable to use it the entire trial. I believe this suspension was a mistake, I am a developer for a living and would greatly appreciate if you could unsuspend my account as I was just trying out Co-Pilot. 

I did not receive any email or notification for the reasoning about the suspension, so as you could imagine it was a big shock to me. I have contacted support already about this, and it has been over a week and I haven't heard anything back. I understand you guys are really busy and must have thousands of tickets to go through. If I need to pay for it I understand and can do so if  that is required to re-instate my account. Again I am so sorry for any confusion with the Co-Pilot free trial, I was not able to access my account to end it. Thank you for whoever has to the time to read this and possibly help me out here.

**Comment by ebndev** [comment]
Hi @thedevinc,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
